sycl/include/sycl/buffer.hpp

```diff
@@ -535,11 +539,16 @@ class buffer : public detail::buffer_plain,
   template <access::mode mode>
   __SYCL2020_DEPRECATED("get_access for host_accessor is deprecated, please "
                         "use get_host_access instead")
-  accessor<T, dimensions, mode, access::target::host_buffer,
-           access::placeholder::false_t, ext::oneapi::accessor_property_list<>>
-  get_access(
-      range<dimensions> accessRange, id<dimensions> accessOffset = {},
-      const detail::code_location CodeLoc = detail::code_location::current()) {
+  accessor<
+      T, dimensions, mode, access::target::host_buffer,
+      access::placeholder::false_t,
+      ext::oneapi::
+          accessor_property_list<>> get_access(range<dimensions> accessRange,
+                                               id<dimensions> accessOffset = {},
+                                               const detail::code_location
+                                                   CodeLoc =
+                                                       detail::code_location::
+                                                           current()) {
```

Comment:
I assume this is just clang-format having changed its mind since last it was run on this?

Reply:
yes, it is